src/ci/docker/x86_64-gnu-llvm-7/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   file \
   curl \
   ca-certificates \
-  python2.7 \
+  python3 \
```

Comment:
Seems like this should still be python2.7 as we're not changing this builder over to python 3.

Reply:
And we're using python2.7 explicitly so that's even worse! Reverting this change to python2.7.

Reply:
`python2.7` package doesn't provide `/usr/bin/python`.
I'm not sure if this PR will build without `python` package as it provides `/usr/bin/python` symlink to `/usr/bin/python2.7` on Ubuntu releases used by CI.

Reply:
Hm, is there a python 2 package providing python?

Reply:
`apt install python` provides the symlink and would also pull `python2.7` as a dependency.

Reply:
Sorry for late reply, cuviper is right.
Also for python 3 we'd have to create symlink by hand or via `update-alternatives` tool.